scripts/dotnet-cli-build/PublishTargets.cs

```diff
@@ -47,7 +47,7 @@ public static BuildTargetResult Publish(BuildTargetContext c)
         [Target(
             nameof(PublishTargets.PublishVersionBadge),
             nameof(PublishTargets.PublishSdkInstallerFile),
-            //nameof(PublishTargets.PublishDebFileToDebianRepo),
+            nameof(PublishTargets.PublishDebFilesToDebianRepo),
```

Comment:
Can we uncomment `BuildPlatform.Ubuntu` for `PublishSdkInstallerFile`?  I see that `PublishDebFilesToDebianRepo` is no longer dependent on `PublishSdkInstallerFile`, so perhaps it's not necessary yet.